BlobForwarder/README.md

```diff
@@ -1,6 +1,34 @@
+
+## Installation
+This integration requires both a New Relic and Azure account.
+
+### Install Using Azure Portal
+Retrieve your [New Relic License Key](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key).
+Then click the button below to start the installation process via the Azure Portal.
+
+[![Deploy to Azure](https://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fnewrelic%2Fnewrelic-azure-functions%2Fmaster%2FarmTemplates%2Fazuredeploy-blobforwarder.json)
+
+### Azure Application Settings
+Parameters that can be configured in your Azure Resource Manager Template
+
+| Parameter  | Required | Default Value | Description
+|---|---|---|---|
+| New Relic License Key  | yes | `none` | Your New Relic Insights [insert key](https://docs.newrelic.com/docs/insights/insights-api/get-data/query-insights-event-data-api#register) |
+| New Relic Endpoint  |  no | `https://log-api.newrelic.com/log/v1` | New Relic Logs [ingestion endpoint](https://docs.newrelic.com/docs/logs/new-relic-logs/log-api/introduction-log-api#endpoint) |
+| Max Retries To Resend Logs  | no | `3` | Number of times the function will attempt to resend data |
+| Retry Interval  | no | `2000` | Interval between retry attempts in milliseconds |
+| Storage Account Name | yes | `none` | Storage Account Name in which the Blobs are allocated. If a new name is provided, a new Storage Account will be created. More information about Storage Account in [azure official documentation](https://docs.microsoft.com/en-us/azure/storage/). |
+| Storage Account Redundancy | yes | `Standard_LRS` | The data in your Azure storage account is always replicated to ensure durability and high availability. Choose a replication strategy that matches your existing storage account or a new one to be created. More information about Storage Account Redundancy in [azure official documentation](https://docs.microsoft.com/en-us/azure/storage/common/storage-redundancy). |
+| Storage Account Kind | yes | `StorageV2` | Indicates the type of storage account. Each type supports different features and has its own pricing model. Consider these differences before you create a storage account to determine the type of account that's best for your applications or choose one that match your existing storage account. More information about Storage Account Kind in [azure official documentation](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-overview#types-of-storage-accounts). |
+| Storage Account Location | yes | `none` | Location where the storage account resides. |
+| Storage Account Container | yes | `none` | Container name inside the Storage Account. More information about Blob storages in [azure official documentation](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction). |
+
```

Comment:
Not sure if this make sense but, I would prefer to order the parameters by 'required'. Up to You